monitor/monitor.c

```diff
@@ -908,7 +907,9 @@ int main (int argc, char **argv)
         break;
     case MODE_AGENT:
         ret = system(SCRIPT_SYSCTL);
-
+        if (ret != 0) {
+            debug("Sysctl -p failed rc=%d.\n", ret);
+        }
```

Comment:
Can we continue with the syscall failure? Especially the variables assignment below.

Reply:
I think 'sysctl -p' is to Load  in  sysctl  settings  from  the  file specified or /etc/sysctl.conf if none given, sometime if customer does not give enough permission it can fail, but it is not a fatal error to stop whole agent(enforcer) process from being run.

Reply:
Talked with George. This is a non-fatal error but will have some implications on performance or capability. We will add more information here (and the same with the change below).

Reply:
I have added more debug message.

Reply:
Maybe something along the line of
```
WARNING: sysctl failed to load /etc/sysctl.conf (including net.core.somaxconn and net.unix.max_dgram_qlen).
It might have a performance implication on the system.
rc = %d.\n
```

Same below.

Reply:
I have updated debug messages in both places, thanks